assignment1.txt

```diff
@@ -0,0 +1,16 @@
+1. Because of the standards, the code requires a return number to tell the program that the code runs smoothly and has no flaws, this helps further execute the code. If using the standard, return 0 is needed, because main is recognised as an integer. However, in this version, because the gcc is able to compile and give some leway, retun 0 isn't required in my version.
+2. I think that %d doesn't make sense because % symbol is used for remainder and there should be one number dividing by the other in other to find the remainder of the mathematical equation.**?
```

Comment:
Hm, not quite! Think about how % can be used in the context of printing. There are also a couple more errors you're missing.